build-support/bower.json

```diff
@@ -10,7 +10,6 @@
   "dependencies": {
     "mocha": "~2.2.4",
     "chai": "~2.3.0",
-    "ember-mocha-adapter": "~0.3.1"
```

Comment:
this unfortunately results in a syntax error since I forgot to remove the trailing comma. this will be fixed by #122